.gitignore

```diff
@@ -1 +1,2 @@
 *.pyc
+*.DS_Store
```

Comment:
you also have to explicitly remove the .DS_Store file from git (git rn .DS_Store)